.ci/openshift-ci/cluster/deployments/relable_selinux.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: restorecon-daemonset
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: restorecon
+  template:
+    metadata:
+      labels:
+        app: restorecon
+    spec:
+      serviceAccountName: kata-deploy-sa
+      hostPID: true
+      containers:
+        - name: restorecon-container
+          image: alpine
+          securityContext:
+            privileged: true
+          command: ["/bin/sh", "-c", "nsenter --target 1 --mount bash -c \"ls -alZ /opt/kata/bin; semanage fcontext -a -t bin_t '/(.*/)?opt/kata/bin(/.*)?'; semanage fcontext -a -t bin_t '/(.*/)?opt/kata/libexec(/.*)?'; semanage fcontext -a -t bin_t '/(.*/)?opt/kata/runtime-rs/bin(/.*)?'; restorecon -Rv /opt/kata; ls -alZ /opt/kata/bin\"; sleep infinity"]
```

Comment:
@ldoktor I think you will need to pass `-e` to `/bin/sh` so to immediately exit error on `nsenter` failure. Otherwise it will always `sleep` and exit success.

Reply:
Nit: loop through the directories names, like:

```
for dir in bin libexec runtime-rs/bin; do echo semanage fcontext -a -t bin_t '/(.*/)?opt/kata/'$dir'(/.*)?'; done
```

Reply:
As for the return code, we don't really care as the pod might be executed multiple times so IMO best effort way should do. Note we might be running on system without selinux completely and in such case it should also succeed.